client/src_editor/Editor_Blocks_Utils.re

```diff
@@ -10,7 +10,7 @@ let renderErrorIndicator = (colStart, colEnd, content) =>
   ++ "\n"
   ++ content;
 
-let executeRessultToWidget = (result: list(Worker_Types.blockData)) => {
+let executeResultToWidget = (result: list(Worker_Types.blockData)) => {
```

Comment:
I blame @-jaredly for this. His reason-language-server is so good that I just tab tab tab